doc/pick_place/pick_place_tutorial.rst

```diff
@@ -43,4 +43,8 @@ The Entire Code
 ---------------
 The entire code can be seen :codedir:`here <pick_place>` in the moveit_tutorials GitHub project.
 
-.. tutorial-formatter:: ./src/pick_place_tutorial.cpp
\ No newline at end of file
+.. |br| raw:: html
```

Comment:
Is this just for a line break? Seems like overkill

Reply:
I couldn't find a better way to do this and it looks real shabby without it.